spec/models/furima_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe Furima, type: :model do
+  
+  before do
+    @furima = FactoryBot.build(:furima)
+  end
+
+  describe "Furima新規登録" do
+    context "登録できるとき" do
+      it "Furima登録ができるとき" do
+        expect(@furima).to be_valid
+      end
+    end
+    context "登録できないとき" do
+      it "imageが空では登録できない" do
+        @furima.image = ''
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Image can't be blank")
+      end
+      it "titleが空では登録できない" do
+        @furima.title = ''
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Title can't be blank")
+      end
+      it "explanationが空では登録できない" do
+        @furima.explanation = ''
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Explanation can't be blank")
+      end
+      it "category_idが空では登録できない" do
+        @furima.category_id = ''
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Category can't be blank")
+      end
+      it "category_idが1では登録できない" do
+        @furima.category_id = 1
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Category can't be blank")
+      end
+      it "status_idが空では登録できない" do
+        @furima.status_id = ''
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Status can't be blank")
+      end
+      it "status_idが1では登録できない" do
+        @furima.status_id = 1
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Status can't be blank")
+      end
+      it "shipping_fee_status_idが空では登録できない" do
+        @furima.shipping_fee_status_id = ''
+        @furima.valid?
+        expect(@furima.errors.full_messages).to include("Shipping fee status can't be blank")
+      end
+      it "shipping_fee_status_idが空では登録できない" do
```

Comment:
```suggestion
      it "shipping_fee_status_idが1では登録できない" do
```
「1では登録できない」にしておきましょう！